### DISCUSSION TITLE
Notes or custom fields for accessories & consumables

### DISCUSSION THREAD

**Discussion by pml-jcare** [snipe/snipe-it#10021] [Q&A] [answered]
Hi, is there any way I can add a custom field or notes to accessories & consumables? We have quite a lot of accessories and they are located in various physical locations depending on size and demand. I was looking to use Notes field to specify location where accessories are stored. Alternativelly it would be great if we can add a field / fieldset to accessories & consumables so we can create a field for Detailed location and specify which room / cabinet / drawer set we store those accessories. Can someone advise if this is doable or can this be added? We are using Hosted version of Snipe-IT

**Comment by snipe** [comment, marked as answer]
This is not currently possible, but there's already an issue open for it. https://github.com/snipe/snipe-it/issues/4261

**Comment by snipe** [comment]
(We're trying hard to get this into v6 which should be out in the next month or so, but if not, it would be soon after.)
